t/.pherkin.yaml

```diff
@@ -11,10 +11,10 @@ default:
          screenshots_dir: screens
          logging_dir: logs
          screenshot_events:
-           pre-step: 0
+           pre-step: 1
            post-step: 1
-           pre-scenario: 0
-           post-scenario: 0
+           pre-scenario: 1
+           post-scenario: 1
```

Comment:
I think the original granularity was good enough for regular testing, no? If so, can you switch this back to what it was? (Fixing up the commit, that is)

Reply:
Indeed. Done